charts/network-observability-operator/templates/flows_flowcollector.yaml

```diff
@@ -0,0 +1,144 @@
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: '{{ .Release.Namespace }}'
+  deploymentModel: Direct
+  agent:
+    type: eBPF
+    ebpf:
+      imagePullPolicy: IfNotPresent
+      logLevel: info
+      sampling: 50
+      cacheActiveTimeout: 5s
+      cacheMaxFlows: 100000
+      # Change privileged to "true" on old kernel version not knowing CAP_BPF or when using "PacketDrop" feature
+      privileged: false
+      # features:
+      # - "PacketDrop"
+      # - "DNSTracking"
+      # - "FlowRTT"
+      interfaces: []
+      excludeInterfaces: ["lo"]
+      kafkaBatchSize: 1048576
+      #flowFilter:
+      #  action: Accept
+      #  cidr: 0.0.0.0/0
+      #  protocol: UDP
+      #  sourcePorts: 53
+      #  enable: true
+      metrics:
+        enable: false
+        server:
+          port: 9400
+      # Custom optionnal resources configuration
+      resources:
+        requests:
+          memory: 50Mi
+          cpu: 100m
+        limits:
+          memory: 800Mi
+  processor:
+    imagePullPolicy: IfNotPresent
+    logLevel: info
+    # Change logTypes to "CONVERSATIONS" or "ALL" to enable conversation tracking
+    logTypes: Flows
+    # Append a unique cluster name to each record
+    # clusterName: <CLUSTER NAME>
+    # addZone: true
+    # subnetLabels:
+    #   openShiftAutoDetect: true
+    #   customLabels:
+    #   - cidrs: []
+    #     name: ""
+    metrics:
+      server:
+        port: 9401
+      disableAlerts: []
+      # includeList:
+      #   - "node_ingress_bytes_total"
+      #   - "workload_ingress_bytes_total"
+      #   - "namespace_flows_total"
+      #   - "namespace_drop_packets_total"
+      #   - "namespace_rtt_seconds"
+    # Custom optionnal resources configuration
+    resources:
+      requests:
+        memory: 100Mi
+        cpu: 100m
+      limits:
+        memory: 800Mi
+  loki:
+    enable: true
+    # Change mode to "LokiStack" to use with the loki operator
+    mode: Monolithic
+    monolithic:
+      url: {{ .Values.lokiUrl }}
```

Comment:
is there a way to configure auth as well?

Reply:
yes, I have to check possible options